ras_party/controllers/queries.py

```diff
@@ -174,22 +174,30 @@ def search_businesses(search_query, session):
     :return: list of businesses
     """
     logger.info('Searching businesses by name with search query', search_query=search_query)
-    filters = list()
-    name_filters = list()
-    trading_as_filters = list()
+    if len(search_query) == 11 and search_query.isdigit():
+        logger.info("Query looks like an ru_ref, searching only on ru_ref", search_query=search_query)
```

Comment:
Is it useful to keep logging the query? Should we just tie this together with a context bound to the logger?

Reply:
Agreed, created a bound logger with that field bound to it.